ouroboros-consensus/src/Ouroboros/Consensus/Ledger/Byron.hs

```diff
@@ -426,12 +426,18 @@ instance Condense (Header (ByronBlock cfg)) where
 
 newtype ByronBlockOrEBB cfg = ByronBlockOrEBB
   { unByronBlockOrEBB :: CC.Block.ABlockOrBoundary ByteString
-  } deriving (Eq, Show)
+  } deriving (Eq)
+
+instance (ByronGiven, Typeable cfg) => Show (ByronBlockOrEBB cfg) where
+  show x = show (blockHash x, unByronBlockOrEBB x)
```

Comment:
This makes them un`Read`able, but I guess that was already the case.

Reply:
Can this not be part of the `Condense` instance? I expect you want this `Show` instance for the QC output, can't the use of `Condensed` do the trick?

Reply:
I was looking at the result of traces; they all involved `showTracing`, `showTracers`, `showProtocolTracers` etc around `debugTrace`. I'd much prefer `condense*` -- since I am intending them for human inspection -- but I'm guessing most event types don't instantiate `Condense` yet.